fdbcli/ConfigureCommand.actor.cpp

```diff
@@ -364,6 +365,13 @@ CommandFactory configureFactory(
         "support for the "
         "database. The configuration can be updated ONLY at the time of database creation and once set can't be "
         "updated for the lifetime of the database.\n\n"
+        "exclude=<ADDRESS...>: Sets the addresses in the format of IP:port pair to be excluded during recruitment. "
+        "Note this should be only used the database is unavailable because of the faulty processes that are blocking "
+        "the recovery from completion. The number of addresses should be less than the replica factor.\n\n"
+        "exclude=<ADDRESS...>: Sets the addresses in the format of IP1:port1,IP2:port2 pairs to be excluded during "
```

Comment:
L368 to L370 seems duplicated from L371 to L374

Reply:
Oops. Merge conflicts. Thanks!